### PR TITLE
feat(react-components): help button in toolbar

### PR DESCRIPTION
**Pull request by pramodcog**
#### Type of change

Feat  

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/UX-979

## Description :pencil:

Help Button in toolbar for Navigation

[Figma Link](https://www.figma.com/file/RSZN6BS8VqQgWRFQ7pIXC9/FDX?type=design&node-id=9019-7470&mode=design&t=9X8FyhFZC4O8UGeC-4)

Implemented preview image from storybook (Updated)
image

## How has this been tested? :mag:

In Storybook

## Test instructions :information_source:

`cd react-components && yarn && yarn build && yarn storybook`
Select `Help` button from Toolbar

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/components/RevealToolbar/Help/MenuSection.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { type HelpMenuSectionProps } from './types';
+import {
+  SectionContainer,
+  SectionTitle,
+  SectionSubTitle,
+  InstructionDetail,
+  SectionContent
+} from './elements';
+
+export const MenuSection = ({
```
Comment (haakonflatval-cognite) [inline react-components/src/components/RevealToolbar/Help/MenuSection.tsx:15]: Nice, I think this is a good way of grouping the components :+1: Could we call this just `Section`? I think `Menu` is a bit misleading

File: react-components/src/components/RevealToolbar/HelpButton.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import styled from 'styled-components';
+import { MouseNavigation } from './Help/MouseNavigation';
+import { TouchNavigation } from './Help/TouchNavigation';
+import { KeyboardNavigation } from './Help/KeyboardNavigation';
+
+export const HelpButton = (): ReactElement => {
+  const [helpEnabled, setHelpEnabled] = useState<boolean>(false);
+
+  const showHelp = (): void => {
+    setHelpEnabled(!helpEnabled);
```
Comment (deep-cognite) [inline react-components/src/components/RevealToolbar/HelpButton.tsx:17]:
```suggestion
    setHelpEnabled((prevState) => !prevState);
```

File: react-components/src/components/RevealToolbar/Help/elements.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+export const InstructionDetail = styled(Detail)`
+  font-family: Inter;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: -0.006em;
+  text-align: left;
+  color: ${Colors['text-icon--interactive--disabled--inverted']};
+  white-space: pre-line;
+`;
```
Comment (deep-cognite) [inline react-components/src/components/RevealToolbar/Help/elements.ts:51]: These looks like Cogs.js typography styling, no? Can't we use Cogs.js typography here instead?

File: react-components/src/components/RevealToolbar/Help/MouseNavigation.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        'Click and drag to rotate, and pan the view. Use mouse wheel to zoom the view. Left click to select'
+      }>
+      <NavigationInstructionGrid>
+        <InstructionText>{'Zoom / scroll'}</InstructionText>
+        <InstructionText>
+          {'Rotate'}
+          <br />
+          <InstructionDetail>{'Click+drag'}</InstructionDetail>
+          <br />
+          <br />
```
Comment (deep-cognite) [inline react-components/src/components/RevealToolbar/Help/MouseNavigation.tsx:29]: I haven't seen the use of <br /> in such a long time 😆

File: react-components/src/components/RevealToolbar/Help/MouseNavigation.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+          <InstructionText>
+            {'Select Objects'}
+            <br />
+            <InstructionDetail>{'Click on interactive objects'}</InstructionDetail>
+          </InstructionText>
+        </InstructionText>
+        <StyledMouse />
+        <InstructionText>
+          {'Pan'}
```
Comment (deep-cognite) [inline react-components/src/components/RevealToolbar/Help/MouseNavigation.tsx:38]: Why {'pan'} and not just 
```suggestion
          Pan
```

Comment (deep-cognite) [reply react-components/src/components/RevealToolbar/Help/MouseNavigation.tsx:38]: (not a big issue tho)

**Review by haakonflatval-cognite — APPROVED** [review]
Nice! LGTM :+1:

**Pull request merged**